docs/07_maintainers_view.md

```diff
@@ -1,78 +1,70 @@
+many features that are a bit more "techy" and may require some knowledge about
+code, however in essence NewPipe should be easy to use, even for your average Android user.
+
+1. __Don't add too many special
+  features.__ NewPipe does not have to be an air plane cockpit. Do not try to fill every single niche that might exist. If people want more advanced features, they
+  would use professional tools. If you add too much functionality, you add complexity, and complexity scares away the average user. Focus on NewPipe's scope as a **media player** for the end user, and only as such. 
+2. __Design the UI so that it is easy for the user__. Try to make it comply with
   [material design guidelines](https://material.io/design/guidelines-overview/).
-3. __Don't add to much features__: Think about the Betamax vs. VHS phenomena
-  or the Unix principle of having one program designed for one specific task:
-  If you add to much functionality you add complexity and this is not appealing
-  to the user. Focus on what NewPipe should be, and make it be only that.
+  
 
 ### Bugfixes
 
 ![kde_in_a_nutshell](img/kde_in_a_nutshell.jpg)]
 
-*Disclaimer: This is a meme maybe in real live it is different. Pleas no shit storm.*
+*Disclaimer: This is a meme. Don't take it personally.*
 
- __Always go for Bugfixes__, as the best application with the best features
+ __Always prioritize fixing bugs__, as the best application with the best features
    does not help much if it is broken, or annoying to use. Now if a program
-   is in an early stage it is quite understandable that many things brake. This
-   is one reason why NewPipe still has no 1 in the beginning of its version
+   is in an early stage it is quite understandable that many things break. This
+   is one reason why NewPipe still has no "1" in the beginning of its version
    number.
-   However by now NewPipe is in a stage where there should be a strong focus on
+   By now, NewPipe is in a stage where there should be a strong focus on
    stability.
 
-1. If there are multiple Pull requests open, check the ones with the bugfixes first.
-2. Do not add to much features every version, as every feature will inevitable
-    introduce more bugs. It is quite ok, if PRs stay open for a while (not to long though).
+1. If there are multiple Pull Requests open, check the ones with bugfixes first.
+2. Do not add too many features every version, as every feature will inevitably
+    introduce more bugs. It is OK if PRs stay open for a while, but don't leave them open for too long.
 3. If there are bugs that are stale, or open for a while bump them from time
    to time, so devs know that there is still something left to fix.
-4. Never accept bugs. From my perception the community does not like to fix bugs, this is why you as a maintainer should
-   especially focus on perusing bugs. 
+4. Never merge PRs with known issues. From my perception the community does not like to fix bugs, this is why you as a maintainer should
```

Comment:
Now that we call it a "Maintainers' Section" (i.e., plural), the word `my` should be replaced by `our`, etc.